.github/workflows/tests.yml

```diff
@@ -126,10 +132,34 @@ jobs:
           push: true
           tags: ${{ needs.tag.outputs.image }}
 
+  test-uv-install:
+    name: Test uv installs (CPU)
+    runs-on: ubuntu-latest
+    needs: [lint]
+    strategy:
+      fail-fast: false
+      matrix:
+        extras: ['', 'determined', 'api', 'openai', 'transformers', 'accelerate', 'comet', 'optional']
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Install uv
+        run: |
+          curl -LsSf https://astral.sh/uv/install.sh | sh
```

Comment:
I'd recommend to use the [setup-uv action](https://github.com/astral-sh/setup-uv)